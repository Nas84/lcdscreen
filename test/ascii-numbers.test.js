/**
 * Unit test for the ascii number representation
 */
var numbers = require("../lib/ascii-art-numbers");

 describe("Ascii Number ", () => {

    it("should containts ten values", () => {
        expect(numbers).toMatchSnapshot();
    })
 })