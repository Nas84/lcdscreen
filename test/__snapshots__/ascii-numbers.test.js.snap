// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ascii Number  should containts ten values 1`] = `
Object {
  "0": Array [
    " _ ",
    "| |",
    "|_|",
  ],
  "1": Array [
    "   ",
    "  |",
    "  |",
  ],
  "2": Array [
    " _ ",
    " _|",
    "|_ ",
  ],
  "3": Array [
    " _ ",
    " _|",
    " _|",
  ],
  "4": Array [
    "   ",
    "|_|",
    "  |",
  ],
  "5": Array [
    " _ ",
    "|_ ",
    " _|",
  ],
  "6": Array [
    " _ ",
    "|_ ",
    "|_|",
  ],
  "7": Array [
    " _ ",
    "  |",
    "  |",
  ],
  "8": Array [
    " _ ",
    "|_|",
    "|_|",
  ],
  "9": Array [
    " _ ",
    "|_|",
    "  |",
  ],
}
`;
